<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DSS::ExplorerBar</class>
 <widget class="QDockWidget" name="DSS::ExplorerBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>1052</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>454</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>524287</height>
   </size>
  </property>
  <property name="styleSheet">
   <string/>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFeature::DockWidgetFloatable|QDockWidget::DockWidgetFeature::DockWidgetMovable</set>
  </property>
  <property name="allowedAreas">
   <set>Qt::DockWidgetArea::LeftDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string notr="true">DeepSkyStacker</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>260</width>
     <height>430</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
    <property name="spacing">
     <number>24</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QScrollArea" name="explorerScrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>298</width>
         <height>859</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>814</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="registerAndStack">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string comment="IDC_STACKING_TITLE">Registering and Stacking</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8" stretch="7,3,4">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QFrame" name="frame_1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <property name="spacing">
               <number>4</number>
              </property>
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QLabel" name="openLights">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string comment="IDC_STACKING_OPENFILES">Open picture files...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="openDarks">
                <property name="styleSheet">
                 <string notr="true">margin-left: 30</string>
                </property>
                <property name="text">
                 <string comment="IDC_STACKING_OPENDARKS">dark files...</string>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="openFlats">
                <property name="styleSheet">
                 <string notr="true">margin-left: 30</string>
                </property>
                <property name="text">
                 <string comment="IDC_STACKING_OPENFLATS">flat files...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="openDarkFlats">
                <property name="styleSheet">
                 <string notr="true">margin-left: 30</string>
                </property>
                <property name="text">
                 <string comment="IDC_STACKING_OPENDARKFLATS">dark flat files...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="openBias">
                <property name="styleSheet">
                 <string notr="true">margin-left: 30</string>
                </property>
                <property name="text">
                 <string comment="IDC_STACKING_OPENOFFSETS">offset/bias files...</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Orientation::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Policy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>5</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="openFilelist">
                <property name="text">
                 <string comment="IDC_STACKING_LOADLIST">Open a File List...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="saveFilelist">
                <property name="text">
                 <string comment="IDC_STACKING_SAVELIST">Save the File List...</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Orientation::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Policy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>5</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="clearList">
                <property name="text">
                 <string comment="IDC_STACKING_CLEARLIST">Clear List</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <property name="spacing">
               <number>4</number>
              </property>
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QLabel" name="checkAll">
                <property name="text">
                 <string comment="IDC_STACKING_CHECKALL">Check all</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="checkAbove">
                <property name="text">
                 <string comment="IDC_STACKING_CHECKABOVE">Check above a threshold...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="unCheckAll">
                <property name="text">
                 <string comment="IDC_STACKING_UNCHECKALL">Uncheck all</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <property name="spacing">
               <number>4</number>
              </property>
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QLabel" name="registerChecked">
                <property name="text">
                 <string comment="IDC_REGISTERING_REGISTERCHECKED">Register checked pictures...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="computeOffsets">
                <property name="text">
                 <string comment="IDC_STACKING_COMPUTEOFFSETS">Compute offsets...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="stackChecked">
                <property name="text">
                 <string comment="IDC_STACKING_STACKCHECKED">Stack checked pictures...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="batchStacking">
                <property name="text">
                 <string comment="IDC_STACKING_BATCHSTACK">Batch stacking...</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_9">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="processing">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string comment="IDC_PROCESSING_TITLE">Processing</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="leftMargin">
            <number>11</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>11</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="openPicture">
             <property name="text">
              <string comment="IDC_PROCESSING_OPENFILE">Open picture file...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="copyPicture">
             <property name="text">
              <string comment="IDC_PROCESSING_COPYTOCLIPBOARD">Copy current picture to clipboard</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="doStarMask">
             <property name="text">
              <string comment="IDC_PROCESSING_CREATESTARMASK">Create a Star Mask...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="savePicture">
             <property name="text">
              <string comment="IDC_PROCESSING_SAVEFILE">Save picture to file...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_10">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="options">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string comment="IDC_OPTIONS_TITLE">Options</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="leftMargin">
            <number>11</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>11</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="settings">
             <property name="text">
              <string comment="IDC_OPTIONS_SETTINGS">Settings...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ddpSettings">
             <property name="styleSheet">
              <string notr="true">margin-left: 30</string>
             </property>
             <property name="text">
              <string comment="IDC_OPTIONS_RAWDDPSETTINGS">Raw/FITS DDP Settings...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="loadSettings">
             <property name="styleSheet">
              <string notr="true">margin-left: 30</string>
             </property>
             <property name="text">
              <string comment="IDC_OPTIONS_LOADSETTINGS">Load...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="saveSettings">
             <property name="styleSheet">
              <string notr="true">margin-left: 30</string>
             </property>
             <property name="text">
              <string comment="IDC_OPTIONS_SAVESETTINGS">Save...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="recommendedSettings">
             <property name="styleSheet">
              <string notr="true">margin-left: 30</string>
             </property>
             <property name="text">
              <string comment="IDC_OPTIONS_RECOMMANDEDSETTINGS">Recommended...</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>5</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="about">
             <property name="text">
              <string comment="IDC_ABOUT">About DeepSkyStacker...</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>5</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="help">
             <property name="text">
              <string comment="IDC_CONTEXTHELP">DeepSkyStacker's Help...</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>5</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="keepTracefile">
             <property name="toolTip">
              <string>The trace file is normally written to the DeepSkyStacker directory in the users
&quot;Documents&quot; directory, and is deleted on exit (unless the application crashes).
Tick this box to keep the trace file.</string>
             </property>
             <property name="text">
              <string>Retain the trace file on exit</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_7">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>5</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="enableSounds">
             <property name="text">
              <string>Enable progress sounds</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_8">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>5</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="showClipping">
             <property name="toolTip">
              <string>In the Processing Panel, show pixels below the Black Level as blue and pixels above the White Level as red.</string>
             </property>
             <property name="text">
              <string>Show clipping for Black/White Points</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
